sos/plugins/logs.py

```diff
@@ -96,7 +97,7 @@ def setup(self):
             except ValueError:
                 days = 3
             if self.get_option("all_logs"):
-                since = ""
+                since = self.get_options("since")
```

Comment:
As my other comment notes - I don't see why this needs to be coupled with `--all-logs`.